### PR TITLE
[zh] Sync troubleshooting-kubeadm.md

### PR DESCRIPTION
**Pull request by Sea-n**
There is only 1 PR after last sync (#33688)
```diff
$ ./scripts/lsync.sh content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md
diff --git a/content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md b/content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md
index ac8d89ee3a..7b195a34fd 100644
--- a/content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md
+++ b/content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md
@@ -114,7 +114,7 @@ sudo kubeadm reset

 A possible solution is to restart the container runtime and then re-run `kubeadm reset`.
 You can also use `crictl` to debug the state of the container runtime. See
-[Debugging Kubernetes nodes with crictl](/docs/tasks/debug-application-cluster/crictl/).
+[Debugging Kubernetes nodes with crictl](/docs/tasks/debug/debug-cluster/crictl/).

```

But there are some missed paragraph and some mis-placed.

### CODE REVIEW

**Comment by jihoon-seo** [comment]
[Netlify build fails](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/62883ea26c697f00081196ee#L77).

After #33861 gets merged, this will be resolved.
So, you may need to rebase after #33861 gets merged.

File: content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md [hunk 1]
```diff
@@ -33,12 +34,73 @@ If your problem is not listed below, please follow the following steps:
...
+-->
+## 由于缺少 RBAC，无法将 v1.18 Node 加入 v1.17 集群
+
+<!--
+In v1.18 kubeadm added prevention for joining a Node in the cluster if a Node with the same name already exists.
+This required adding RBAC for the bootstrap-token user to be able to GET a Node object.
+
+However this causes an issue where `kubeadm join` from v1.18 cannot join a cluster created by kubeadm v1.17.
+-->
+自从 v1.18 后，如果集群中已存在同名 Node，kubeadm 将防止 Node 加入集群。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:54]:
```suggestion
自从 v1.18 后，如果集群中已存在同名 Node，kubeadm 将禁止 Node 加入集群。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md [hunk 2]
```diff
@@ -33,12 +34,73 @@ If your problem is not listed below, please follow the following steps:
...
+To workaround the issue you have two options:
+
+Execute `kubeadm init phase bootstrap-token` on a control-plane node using kubeadm v1.18.
+Note that this enables the rest of the bootstrap-token permissions as well.
+
+or
+
+Apply the following RBAC manually using `kubectl apply -f ...`:
+-->
+要解决此问题，您有两种选择：
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:69]:
```suggestion
要解决此问题，你有两种选择：
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md [hunk 3]
```diff
@@ -33,12 +34,73 @@ If your problem is not listed below, please follow the following steps:
...
+
+使用 kubeadm v1.18 在控制平面节点上执行 `kubeadm init phase bootstrap-token`。
+请注意，这也会启用其余的 bootstrap-token 权限。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:72]:
```suggestion
请注意，这也会启用 bootstrap-token 的其余权限。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md [hunk 4]
```diff
@@ -33,12 +34,73 @@ If your problem is not listed below, please follow the following steps:
...
+
+或是也可以使用 `kubectl apply -f ...` 手动应用以下 RBAC：
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:74]:
```suggestion
或者，也可以使用 `kubectl apply -f ...` 手动应用以下 RBAC：
```

**Review by tengqm — COMMENTED** [review]
Thanks.
/approve

**Comment by tengqm** [comment]
@Sea-n pls rebase this onto main branch and commit again.

**Pull request merged**